effektio_flutter_sdk/example/.flutter-plugins

```diff
@@ -0,0 +1,15 @@
+# This is a generated file; do not edit or check into version control.
```

Comment:
why are you re-adding the `example`-folder? I removed that on purpose as it is useless for us for now.